x-pack/test/fleet_api_integration/apis/epm/install_hidden_datastreams.ts

```diff
@@ -19,83 +19,14 @@ export default function (providerContext: FtrProviderContext) {
     await supertest.delete(`/api/fleet/epm/packages/${name}/${version}`).set('kbn-xsrf', 'xxxx');
   };
 
-  // Failing: See https://github.com/elastic/kibana/issues/184310
-  describe.skip('installing with hidden datastream', async () => {
+  describe('installing with hidden datastream', async () => {
     skipIfNoDockerRegistry(providerContext);
     setupFleetAndAgents(providerContext);
 
     afterEach(async () => {
       await deletePackage('apm', '8.8.0');
     });
 
-    it('should rollover hidden datastreams when failed to update mappings', async function () {
```

Comment:
The update between the two fixture versions of APM we maintain in the FTR directory no longer triggers a mapping conflict, likely because of upstream changes in Elasticsearch. Rather than try to find a new means to trigger a mapping conflict here, I think removing this in favor of the existing test coverage in Fleet's unit tests is sufficient.